pkg/lib/cockpit-components-firewalld-request.jsx

```diff
@@ -30,7 +30,7 @@ const _ = cockpit.gettext;
 const firewalld = cockpit.dbus('org.fedoraproject.FirewallD1', { superuser: "try" });
 
 function debug() {
-    if (window.debugging == "all" || window.debugging == "firewall")
+    if (window.debugging == "all" || window.debugging?.includes("firewall"))
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18492-20230313-062223-0ed5c399-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)